rust/xain-proto/generate.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -e
+cd "$(dirname "$0")"
+
+if ! [[ -x "$(command -v grpc_rust_plugin)" ]]; then
+    echo "Error: grpc_rust_plugin was not found"
+    echo
+    echo "To install, run: cargo install grpcio-compiler"
+    exit 1
+fi
+
+proto_files="
+../../numproto/numproto/protobuf/ndarray.proto
```

Comment:
This file does not exist in this repo. Is this something that we need to manually add before being able to compile the proto files?

If so it would be helpful to have a README under the `rust/` folder detailing the steps to build the rust side.

Reply:
Ooops, I just cloned the `numproto` repository manually. Now I've added it as a submodule to this repository and the `generate.sh` script will update that submodule before trying to generate Rust bindings for `ndarray.proto`.